src/controller.cpp

```diff
@@ -723,7 +723,10 @@ int Controller::import_opml(const std::string& opmlFile,
 	auto urlReader = FileUrlReader(urlFile);
 	const auto error_message = urlReader.reload(); // Load existing URLs
 	if (error_message.has_value()) {
-		std::cout << error_message.value() << std::endl;
+		std::cout << strprintf::fmt(
+				_("An error occured while importing OPML to url file %s : %s"),
```

Comment:
```suggestion
				_("Error importing OPML to urls file %s: %s"),
```

This is shorter, giving more space for paths and the clarification; it also uses terminology and punctuation the rest of Newsboat does ;)